typings/index.d.ts

```diff
@@ -2071,17 +2076,19 @@ declare module 'discord.js' {
     delete(reason?: string): Promise<void>;
     edit(options: WebhookEditData): Promise<Webhook>;
     send(
-      content?: StringResolvable,
-      options?: (WebhookMessageOptions & { split?: false }) | MessageAdditions,
+      content: APIMessageContentResolvable | (WebhookMessageOptions & { split?: false }) | MessageAdditions,
     ): Promise<Message>;
+    send(options: WebhookMessageOptions & { split: true | SplitOptions }): Promise<Message[]>;
+    send(options: WebhookMessageOptions | MessageAdditions | APIMessage): Promise<Message | Message[]>;
```

Comment:
```suggestion
    send(options: APIMessage): Promise<Message | Message[]>;
```
the overloads above this handle `WebhookMessageOptions | MessageAdditions`

Reply:
I've removed `MessageAdditions` from that overload, but `WebhookMessageOptions` should stay. This ensures this is allowed:

```ts
const options: WebhookMessageOptions = {}
await webhook.send(options) // type Message | Message[]
```

If it wasn't there, this would happen because the type of `options.split` is `boolean | SplitOptions` and does not match any of the overloads:

```
No overload matches this call.
  Overload 1 of 6, '(content: string | number | bigint | boolean | symbol | readonly any[] | MessageEmbed | MessageAttachment | (MessageEmbed | MessageAttachment)[] | (WebhookMessageOptions & { ...; })): Promise<...>', gave the following error.
    Argument of type 'WebhookMessageOptions' is not assignable to parameter of type 'string | number | bigint | boolean | symbol | readonly any[] | MessageEmbed | MessageAttachment | (MessageEmbed | MessageAttachment)[] | (WebhookMessageOptions & { ...; })'.
      Type 'WebhookMessageOptions' is not assignable to type 'WebhookMessageOptions & { split?: false | undefined; }'.
        Type 'WebhookMessageOptions' is not assignable to type '{ split?: false | undefined; }'.
          Types of property 'split' are incompatible.
            Type 'boolean | SplitOptions | undefined' is not assignable to type 'false | undefined'.
              Type 'true' is not assignable to type 'false | undefined'.
  Overload 2 of 6, '(options: WebhookMessageOptions & { split: true | SplitOptions; }): Promise<Message[]>', gave the following error.
    Argument of type 'WebhookMessageOptions' is not assignable to parameter of type 'WebhookMessageOptions & { split: true | SplitOptions; }'.
      Type 'WebhookMessageOptions' is not assignable to type '{ split: true | SplitOptions; }'.
        Types of property 'split' are incompatible.
          Type 'boolean | SplitOptions | undefined' is not assignable to type 'true | SplitOptions'.
            Type 'undefined' is not assignable to type 'true | SplitOptions'.
  Overload 3 of 6, '(options: APIMessage): Promise<Message | Message[]>', gave the following error.
    Argument of type 'WebhookMessageOptions' is not assignable to parameter of type 'APIMessage'.
      Type 'WebhookMessageOptions' is missing the following properties from type 'APIMessage': data, isUser, isWebhook, options, and 5 more.
```